account/account.go

```diff
@@ -68,19 +68,11 @@ func GetAccount(ctx context.Context, cfg *config.Configuration, fetcher stored_r
 				}}
 			}
 		}
-		usingGDPRChannelEnabled := useGDPRChannelEnabled(account)
-		usingCCPAChannelEnabled := useCCPAChannelEnabled(account)
 
-		if usingGDPRChannelEnabled {
-			me.RecordAccountGDPRChannelEnabledWarning(accountID)
-		}
-		if usingCCPAChannelEnabled {
-			me.RecordAccountCCPAChannelEnabledWarning(accountID)
-		}
 		for _, purposeName := range deprecatedPurposeFields {
 			me.RecordAccountGDPRPurposeWarning(accountID, purposeName)
 		}
-		if len(deprecatedPurposeFields) > 0 || usingGDPRChannelEnabled || usingCCPAChannelEnabled {
+		if len(deprecatedPurposeFields) > 0 {
```

Comment:
I know that this line gets removed in another Remove Config PR, but it leaves me wondering if the `RecordAccountUpgradeStatus` metric should be removed as well?

Reply:
👍 merged with master to resolve conflicts and removed `RecordAccountUpgradeStatus` in the process.